### DISCUSSION TITLE
Convert msh format to Gamma Mesh format

### DISCUSSION THREAD

**Discussion by J-AKAZ** [neperfepx/neper#111] [General]
Hello dears,
I would like to know if you have ever programmed any code to convert .gmsh files to .meshb files (Gamma Mesh Format), or there is a way to do that?
In fact, the Gamma Mesh Format (GMF) and the associated library libMeshb provide programers of simulation and meshing software with an easy way to store their meshes and physical solutions.
You can get the documentation in: https://github.com/LoicMarechal/libMeshb.
Thank you

**Comment by rquey** [comment]
I haven't. Maybe others have.

Are there specifications for this mesh format somewhere? I did not find anything in the doc.

It is generally not complicated to create a procedure that writes the mesh at a new format; see `src/neut/neut_mesh/neut_mesh_fprintf`.
